### PR TITLE
Qt controls gui combined

### PR DESCRIPTION
**Pull request by ayambabu23**
Combined controls and GUI in PyQt.

### CODE REVIEW

File: .gitignore
Comment (lutetjeff) [inline .gitignore]: move this line below the comment on line 2

File: gui.py
Comment (priime0) [inline gui.py]: Style: two blank lines between imports.

File: gui.py
Comment (lutetjeff) [inline gui.py]: probably better as a file-scope constant.

File: gui.py
Comment (lutetjeff) [inline gui.py]: same here, probably better as a file-scope constant.

File: gui.py
Comment (lutetjeff) [inline gui.py]: maybe replace this with a `thruster_speed: list[int]`

File: gui.py
Comment (priime0) [inline gui.py]: Remove unnecessary comments here.

File: serialtext.py
Comment (priime0) [inline serialtext.py]: Is there a need to commit this file?

File: gui.py
Comment (priime0) [inline gui.py]: Move this file into the `controls-pyqt` directory.

File: gui.py
Comment (lutetjeff) [inline gui.py]: please, all constants belong on the file scope, not a class method!

File: gui.py
Comment (lutetjeff) [inline gui.py]: modularize this chunk, between lines 81 and 147. use private methods (`__method`)

File: gui.py
Comment (priime0) [inline gui.py]: Create variables for `self.mcu.latest_voltage`, `self.mcu.latest_gyro`, etc. and use those.

File: gui.py
Comment (priime0) [inline gui.py]: For each block in this function, create a private function (prefix with `__`) whose name summarizes what the block of code does.

File: gui.py
Comment (lutetjeff) [inline gui.py]: clean up; add `mcu.close_serial()` to avoid port blockage

File: gui.py
Comment (priime0) [inline gui.py]: Change to `update_text`. If it is also a private function, prefix it with `__`.

File: serialtext.py
Comment (lutetjeff) [inline serialtext.py]: What is this file for?

File: gui.py
Comment (priime0) [inline gui.py]: Change to `setup_ui`.

**Review by lutetjeff — CHANGES_REQUESTED** [review]
Mostly just formatting changes, but make sure you have docstrings on each method that should be used or seen, and private (`__`) all other methods.

File: gui.py
Comment (priime0) [inline gui.py]: Use `snake_case` for `keyPressEvent`, `keyReleaseEvent`, and `closeEvent`.

**Review by priime0 — CHANGES_REQUESTED** [review]

File: controls-pyqt/exit_program.py
Comment (priime0) [inline controls-pyqt/exit_program.py:9]: Would `comms.kill_op()` include `mcu.close_serial()`?

Comment (ayambabu23) [reply controls-pyqt/exit_program.py:9]: It can.

**Review by priime0 — APPROVED** [review]
lgtm 👍 . Wait for @lutet88 to review before the merge.

**Review by lutetjeff — APPROVED** [review]
Not quite ideal, but totally ready to merge.

**Pull request merged**
